backend/src/main/java/c/team/configuration/MongoConvertersConfig.java

```diff
@@ -0,0 +1,36 @@
+package c.team.configuration;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.core.convert.converter.Converter;
+import org.springframework.data.mongodb.core.convert.MongoCustomConversions;
+
+import java.time.OffsetDateTime;
+import java.time.ZoneOffset;
+import java.util.List;
+
+@Configuration
+public class MongoConvertersConfig {
+
+    @Bean
+    public MongoCustomConversions mongoCustomConversions() {
+        return new MongoCustomConversions(List.of(
+                new OffsetDateTimeReadConverter(),
+                new OffsetDateTimeWriteConverter()
+        ));
+    }
+
+    static class OffsetDateTimeWriteConverter implements Converter<OffsetDateTime, String> {
+        @Override
+        public String convert(OffsetDateTime source) {
+            return source.toInstant().atZone(ZoneOffset.UTC).toString();
+        }
+    }
+
+    static class OffsetDateTimeReadConverter implements Converter<String, OffsetDateTime> {
+        @Override
+        public OffsetDateTime convert(String source) {
+            return OffsetDateTime.parse(source);
+        }
+    }
+}
```

Comment:
O nice, czy to odpowiada za konwersję string <-> timestamp?

Reply:
niee, mongodb nie ogarnialo wpisywania do bazy danych typu `OffsetDateTime`, więc trzeba było dodać taki converter